### ISSUE TITLE
Document supported python version and allow 3.7

### ISSUE THREAD

**Issue by tuukkamustonen** [MousaZeidBaker/aws-lambda-typing#8]
It would be nice for the README to specify which python versions are supported.

Also, 3.7 is now not supported. Is that because of missing `Literal`, `TypedDIct` or something? Could it be made supported?

**Comment by MousaZeidBaker** [comment]
Hi @tuukkamustonen, thanks for opening this issue. 

I've added a badge that shows supported python versions in [PR9](https://github.com/MousaZeidBaker/aws-lambda-typing/pull/9).

Regarding supporting <3.8, you are much right that [TypedDIct](https://docs.python.org/3/library/typing.html#typing.TypedDict), [Literal](https://docs.python.org/3/library/typing.html#typing.Literal) as well as [Final](https://docs.python.org/3/library/typing.html#typing.Final) were introduced in 3.8 so unless they can be replaced we need to stick with 3.8.

**Comment by MousaZeidBaker** [comment]
I was looking at the [typing](https://github.com/python/typing) source code and came across [typing_extensions](https://github.com/python/typing/blob/master/typing_extensions/README.rst), a module that contains backports of new types and makes them available for Python >= 3.5. I'll give it a try.

**Comment by tuukkamustonen** [comment]
Thanks for the quick actions!

Yeah, `typing_extensions` is sort of a incubator for new features. If you include it conditionally (with `< 3.8` python) that should do it. Though, the drawback is that you have to either import these features from `typing_extensions` or have conditional imports like:

```py
try:
    from typing import TypedDict
except ImportError:
    from typing_extensions import TypedDict
```

...so it's pretty annoying and I understand if you decide to pass. (Unless there's a better way, of course 😄.)
